static/app/views/monitors/components/processingErrors/monitorProcessingErrors.tsx

```diff
@@ -51,45 +65,70 @@ export default function MonitorProcessingErrors({
     </StyledHovercard>
   );
 
-  const [expanded, setExpanded] = useState(-1);
-  const accordionErrors = Object.values(errorsByType).map((errors, index) => {
-    const isExpanded = expanded === index;
-    return (
-      <ErrorGroup key={index}>
-        <ErrorHeader>
-          <Tag type="error">{errors.length}x</Tag>
-          <ProcessingErrorTitle type={errors[0].error.type} />
-
-          <Button
-            icon={<Chevron size="small" direction={isExpanded ? 'up' : 'down'} />}
-            aria-label={isExpanded ? t('Collapse') : t('Expand')}
-            aria-expanded={isExpanded}
-            size="zero"
-            borderless
-            onClick={() => setExpanded(isExpanded ? -1 : index)}
-          />
-        </ErrorHeader>
-        {isExpanded && (
-          <List symbol="bullet">
-            {errors.map(({error, checkin}, errorIndex) => (
-              <ListItem key={errorIndex}>
-                <ProcessingErrorItem
-                  error={error}
-                  checkinTooltip={renderCheckinTooltip(checkin)}
-                />
-              </ListItem>
-            ))}
-          </List>
-        )}
-      </ErrorGroup>
-    );
-  });
+  const showingMultipleProjects = Object.keys(errorsByProjectByType).length > 1;
+  const [expanded, setExpanded] = useState('');
+  const accordionErrors = Object.entries(errorsByProjectByType).map(
+    ([projectId, errorsByType]) => {
+      const project = projects.find(({id}) => id === projectId);
```

Comment:
this is definitely not ideal, and feeling a need for a ids->projects type of functionality within useProjects :(